src/pages/evses/variable-attributes/VariableAttributes.ts

```diff
@@ -10,32 +10,32 @@ import {
   DataEnumType,
```

Comment:
I think we no longer need this `src/pages/evses/variable-attributes` files right since we moved them to `src/pages/variable-attributes` right? Can we safely remove these? 🙏